content/06.methods.md

```diff
@@ -183,54 +182,54 @@ We filtered out any CNVs that overlapped 50% or more with immunoglobulin, telome
 
 #### Somatic Structural Variant Calling (WGS samples only)
 
-We used Manta [@doi:10/gf3ggb] for structural variant (SV) calls, and we limited to regions used in Strelka2.
+We used `Manta` [@doi:10/gf3ggb] for structural variant (SV) calls, and we limited to regions used in `Strelka2`.
 The hg38 reference for SV calling used was limited to canonical chromosome regions.
 The somatic DNA workflows for SNV, indel, copy number, and SV calling can be found in the [OpenPBTA workflows Github repository](https://github.com/d3b-center/OpenPBTA-workflows/).
-We used [AnnotSV v2.1](https://lbgi.fr/AnnotSV/) [@doi:10.1093/bioinformatics/bty304] to annotate Manta output.
+We used `AnnotSV` [@doi:10.1093/bioinformatics/bty304] to annotate `Manta` output.
 This associated workflow is available in the [workflows GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/).
```

Comment:
Maybe we could save some space by changing this sentence, and delete line 187? For example, 

"All associated workflows described here are available in the [workflows GitHub repository](https://github.com/d3b-center/OpenPBTA-workflows/)."

Reply:
We do have unlimited space in methods, but I did update since it was redundant :)